ProjectCoimbra.UWP/Project.Coimbra/Pages/LaneSettingsPage.xaml.cs

```diff
@@ -196,17 +196,29 @@ private void OptimizeKeyboard_Click(object sender, RoutedEventArgs e)
 
             this.Lanes.Children.Clear();
 
+            const int newLanes = 7;
```

Comment:
It's a matter of taste really (and therefore entirely up to you), but you could remove this and then:
- Initialise `keyboardKeys` with just the elements, which will give it an implied length of 7.
- You can then replace all instances of `newLanes` with `keyboardKeys.Length`.